cmd/detect/definitions.go

```diff
@@ -39,6 +39,7 @@ const (
 	taskFroniusWeb   = "fronius-web"
 	taskTasmota      = "tasmota"
 	taskShelly       = "shelly"
+	taskPulsatrix    = "pulsatrix"
```

Comment:
Does that detect anything? I don't think we should add an "open port 80" detection here.